zoom-scratch.coffee

```diff
@@ -0,0 +1,74 @@
+# Description:
```

Comment:
Was this file intended to be added twice? Seems to be in root and the (more proper) `scripts/` dir too

Reply:
oops! thx.  fixed in b19f161